internal/core/export/adt.go

```diff
@@ -329,7 +329,7 @@ func (e *exporter) resolve(env *adt.Environment, r adt.Resolver) ast.Expr {
 		// comprehensions originate from a single source and do not need to be
 		// handled.
 		if env != nil { // for generated stuff
-			if v := env.Vertex; !v.IsDynamic {
+			if v := env.Vertex; v != nil && !v.IsDynamic {
```

Comment:
Spoke to Marcel briefly about this fix; we think that `env.Vertex` should never be nil, but we haven't found the time to understand the root cause, so we're happy with this fix in the meantime since it's better than nothing and no other tests are broken by it.

It would still be nice to add a TODO, like:

    // TODO: note that env.Vertex should never be nil; investigate and replace the nil check below.

Reply:
We agreed I would do this when importing the PR into Gerrit.